app/assets/javascripts/components/autocomplete.js

```diff
@@ -85,6 +83,10 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
         if (options.length) {
           options[0].selected = true
         }
+
+        if ($insertButton && value) {
```

Comment:
This looks very specific to the behaviour for contacts - is there something we could do to call this out at a higher level? I'm also wondering if we should be using the translation file to render the snippet here? Sorry for not mentioning this the first time - I think I've started avoiding subconciously the autocomplete ;-).

Reply:
Renamed the init function to `initAutoCompleteContacts` to reflect its used case and using the translation file for the snippet template via a `data` attribute.